xplique/attributions/forgrad.py

```diff
@@ -0,0 +1,90 @@
+"""
+Module related to ForGrad method enchancement
+"""
+
+from functools import lru_cache
+
+import tensorflow as tf
+import numpy as np
+
+
+@lru_cache(maxsize=32)
+def _low_pass_real_signal_mask(size : int, bandwith : int) -> tf.Tensor:
+    """
+    Create a low pass filter mask to be applied on a real signal.
+    Since the DFT of a real signal is Hermitian-symmetric, only returns the fft_length / 2 + 1
+    unique components of the FFT.
+
+    Parameters
+    ----------
+    size
+        Size of the mask.
+    bandwith
+        Bandwith of the low pass filter. The higher the bandwith, the more frequencies are kept.
+
+    Returns
+    -------
+    mask
+        Low pass filter mask.
+    """
+    center = (int(size/2), int(size/2))
+
+    y_grid, x_grid = np.ogrid[:size, :size]
+    dist_from_center = np.sqrt((x_grid - center[0])**2 + (y_grid-center[1])**2)
+
+    mask = dist_from_center <= bandwith
+    # un-center the mask
+    mask = tf.signal.fftshift(mask)
+    # keep only the unique components
+    mask = tf.cast(mask, tf.float32)[:, :size//2 + 1]
+
+    return mask
+
+def forgrad(explanations : tf.Tensor, sigma: int = 15) -> tf.Tensor:
+    """
+    ForGRAD is a method that enhances any attributions explanations (particularly useful on
```

Comment:
As it works on "any attributions explanations" I would have put it in the `commons` folder as we did for the `operators`. I guess it is debatable but in my opinion, it would make sense to be in the `attributions` folder you should inherit from `base` (or another attribution method). But here, it is really a standalone function that we can add "on-top".